composer.json

```
@@ -74,7 +74,7 @@
     "pear/net_url2": "^2.2",
     "phive/twig-extensions-deferred": "^2.0",
     "piwik/device-detector": "^3.9",
-    "presta/sitemap-bundle": "^1.5",
+    "presta/sitemap-bundle": "^2.1",
```

Should we allow both versions to be on the safe side? So somebody who need version 1 can set it to the own composer.json and can downgrade it.

```suggestion
    "presta/sitemap-bundle": "^1.5 || ^2.1",
```